forms/NullableField.php

```diff
@@ -77,53 +100,80 @@ public function getIsNullId() {
 	}
 
 	/**
-	 * (non-PHPdoc)
-	 * @see framework/forms/FormField#Field()
+	 * @param array $properties
+	 *
+	 * @return string
 	 */
 	public function Field($properties = array()) {
-		if ( $this->isReadonly()) {
+		if($this->isReadonly()) {
 			$nullableCheckbox = new CheckboxField_Readonly($this->getIsNullId());
 		} else {
 			$nullableCheckbox = new CheckboxField($this->getIsNullId());
 		}
+
 		$nullableCheckbox->setValue(is_null($this->dataValue()));
 
-		return $this->valueField->Field() . ' ' . $nullableCheckbox->Field()
-			. '&nbsp;<span>' . $this->getIsNullLabel().'</span>';
```

Comment:
[this](https://github.com/silverstripe/silverstripe-framework/pull/4114/files#diff-6e6a6671e3877cda188e546cd9a96624R117)